docs/user-guide/TutorialDemoServer.md

```diff
@@ -10,7 +10,7 @@ Let's create a fresh directory `C:\restler-test` where will run RESTler on this
 
 Let's copy the Swagger spec `swagger.json` from `restler\demo_server` into `C:\restler-test` and let's compile it with RESTler:
 
-`C:\RESTler\restler\Restler.exe compile --api_spec C:\restler-test\swagger.json`
+`C:\restler-test\restler\Restler.exe compile --api_spec C:\restler-test\swagger.json`
```

Comment:
>C:\restler-test\ [](http://example.com/codeflow?start=1&length=16)

The intent here is to have separate directories for the RESTler drop directory (target directory for the built binaries) and the directory where you create the RESTler job, so these should stay different.  For clarification, what do you think about putting a sentence right before the 'Compile' section saying "The below sections assume you have built RESTler, and the RESTler drop is located in the 'C:\restler' directory.

Reply:
In that case, how do you feel about changing the directory to restler_bin? Under Setting up RESTler in the main readme, this is where the bin files are initially created, so it's not clear what directory "restler" refers to. Yes, I think a note under Compile will be useful as clarification in case that step was skipped as well.

Reply:
Yep, sounds good to modify c:\restler to C:\restler_bin (then, please find and replace this directory in all of the other files under 'docs', there are many other usages).

Also, since this is now a bigger change, could you please also change 'c:\restler-test' to be 'c:\demo-server-test' in this tutorial?